test/e2e-common.sh

```diff
@@ -125,6 +125,19 @@ function test_task_creation() {
             cp ${yaml} ${TMPF}
             [[ -f ${taskdir}/tests/pre-apply-task-hook.sh ]] && source ${taskdir}/tests/pre-apply-task-hook.sh
             function_exists pre-apply-task-hook && pre-apply-task-hook
+
+            [[ -d ${taskdir}/tests/fixtures ]] && {
```

Comment:
♥️ , I think the magic needs a bit more explanation as to how this works in the commit message.

Reply:
It is I think in the commit message, but I definitively would update the [CONTRIBUTING](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md#end-to-end-testing) guide on how it works whe we find a consensus on how we want to implement this